package.json

```diff
@@ -92,7 +92,7 @@
     "json-loader": "^0.5.7",
     "node-sass": "^4.12.0",
     "node-ssi": "^0.3.2",
-    "npm": "^6.14.8",
+    "np": "^6.5.0",
```

Comment:
Has this really changed from npm to np?

Reply:
I was using NPM API for publishing the SWE package but recently they have made some changes and also NPM is much bigger in size.

I replaced with NP this is an NPM package and it is for authentication and publishing SWE package, lightweight and easy to use.

Reply:
https://www.npmjs.com/package/np is mostly a wrapper for the npm that is already installed.

Reply:
We need to install NPM or NP because we are using API from code, not from CLI. NP provides better integration and it is for just one task that is to publish.

A system installed NPM path can be unstable because the path is different Linux path is different from windows path etc.